src/code/gfxprint.c

```diff
@@ -341,15 +366,17 @@ void GfxPrint_Open(GfxPrint* this, Gfx* dList) {
         this->dList = dList;
         GfxPrint_Setup(this);
     } else {
-        PRINTF("gfxprint_open:２重オープンです\n");
+        GFXPRINT_PRINTF("gfxprint_open:２重オープンです\n");
```

Comment:
I think I would rather just an in-place ifdef that chooses between osSyncPrintf and PRINTF over this custom macro

Reply:
I went with `#if PLATFORM_N64 || OOT_DEBUG`, is that fine?